### PR TITLE
Correct dependencies for CRSF in build options

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: Tools/scripts/build_options.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def __init__(self,
     Feature('Telemetry', 'FrSky', 'AP_FRSKY_TELEM_ENABLED', 'Enable FrSky Telemetry', 0, None),
     Feature('Telemetry', 'FrSky D', 'AP_FRSKY_D_TELEM_ENABLED', 'Enable FrSkyD Telemetry', 0, 'FrSky'),
     Feature('Telemetry', 'FrSky SPort', 'AP_FRSKY_SPORT_TELEM_ENABLED', 'Enable FrSkySPort Telemetry', 0, 'FrSky'),  # noqa
-    Feature('Telemetry', 'FrSky SPort PassThrough', 'AP_FRSKY_SPORT_PASSTHROUGH_ENABLED', 'Enable FrSkySPort PassThrough Telemetry', 0, 'FrSky SPort'),  # noqa
+    Feature('Telemetry', 'FrSky SPort PassThrough', 'AP_FRSKY_SPORT_PASSTHROUGH_ENABLED', 'Enable FrSkySPort PassThrough Telemetry', 0, 'FrSky SPort, FrSky'),  # noqa
```
Comment (peterbarker) [inline Tools/scripts/build_options.py:68]: So `index2.html` really should be recursively selecting the options here: https://github.com/ardupilot/CustomBuild/blob/main/templates/index2.html#L120 - but it doesn't so we get to this sort of thing.

I'm guessing there might be more like this lurking....

Comment (peterbarker) [inline Tools/scripts/build_options.py:68]:
```suggestion
    Feature('Telemetry', 'FrSky SPort PassThrough', 'AP_FRSKY_SPORT_PASSTHROUGH_ENABLED', 'Enable FrSkySPort PassThrough Telemetry', 0, 'FrSky SPort,FrSky'),  # noqa
```

That space *might* breakingthings....

**Review by peterbarker — CHANGES_REQUESTED** [review]
Need that space removed...

File: Tools/scripts/build_options.py (hunk 1 above)
Comment (Hwurzburg) [reply Tools/scripts/build_options.py]: probably

Comment (Hwurzburg) [reply Tools/scripts/build_options.py]: fixed

File: Tools/scripts/build_options.py [hunk 2]
```diff
@@ -55,8 +55,8 @@ def __init__(self,
     Feature('Ident', 'ADSB_UAVIONX_UCP', 'HAL_ADSB_UCP_ENABLED', 'Enable uAvionix UCP ADSB', 0 , 'ADSB'),
     Feature('Ident', 'AIS', 'AP_AIS_ENABLED', 'Enable AIS', 0, None),
 
-    Feature('Telemetry', 'CRSF', 'HAL_CRSF_TELEM_ENABLED', 'Enable CRSF Telemetry', 0, 'FrSky SPort PassThrough'),
-    Feature('Telemetry', 'CRSFText', 'HAL_CRSF_TELEM_TEXT_SELECTION_ENABLED', 'Enable CRSF Text Param Selection', 0, 'CRSF,OSD_PARAM'),  # NOQA: E501
+    Feature('Telemetry', 'CRSF', 'HAL_CRSF_TELEM_ENABLED', 'Enable CRSF Telemetry', 0, 'FrSky SPort PassThrough,FrSky,FrSky Sport'),  # noqa
```
Comment (andyp1per) [inline Tools/scripts/build_options.py:58]: You need passthrough here also

**Review by andyp1per — CHANGES_REQUESTED** [review]

**Review by andyp1per — APPROVED** [review]

**Review by peterbarker — APPROVED** [review]

**Pull request merged**